projects/worker-metadata/manifest.yml

```diff
@@ -0,0 +1,9 @@
+packages:
+  __APP_PACKAGE__:
+    actions:
+      worker-metadata:
+        function: 'worker-metadata.js'
+        runtime: 'nodejs:10'
```

Comment:
For science - did you also try it with node 12?

Reply:
Yeah, suggest to set all examples to use node 12.

Reply:
I've applied that change.